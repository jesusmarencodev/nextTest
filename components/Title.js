import React from 'react';
import { useSelector, useDispatch } from "react-redux";


export const Title = () => {
  const label ="Hello world";
  const state = useSelector((state) => state);

  console.log(state)
  return (
    <div>
      <h1>{label}</h1>
    </div>
  )
}
