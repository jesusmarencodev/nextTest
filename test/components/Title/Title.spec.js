import { mount, shallow } from 'enzyme';
import { Title } from '../../../components/Title';


describe('Title Component', () => {

  it("el componente existe", ()=>{
    const wrapper = shallow(<Title/>);

    expect( wrapper ).toMatchSnapshot();
  })
  



  it('should render without crashing', () => {
    const wrapper = mount(<Title />);

    expect(wrapper.find('h1')).toHaveLength(1);
  });
});