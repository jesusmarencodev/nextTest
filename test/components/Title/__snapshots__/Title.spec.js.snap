// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component el componente existe 1`] = `ShallowWrapper {}`;
